cron/poll_read_issue_comments.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import arrow
+import json
+import os
+from os.path import join, abspath, dirname
+
+import settings
+import github_api as gh
+
+THIS_DIR = dirname(abspath(__file__))
+# Hopefully this isn't overwritten on pulls
+SAVED_COMMANDS_FILE = join(THIS_DIR, '..', "server/issue_commands_ran.json")
+if not os.path.exists(SAVED_COMMANDS_FILE):
+    '''
+        "comment_ids_ran": [
+            {
+                "comment_id" : ..,
+                "chaos_response_id" : ..
+            }
+        ]
+    '''
+    with open(SAVED_COMMANDS_FILE, 'w') as f:
+        dummy_data = {"comment_ids_ran": []}
+        json.dump(dummy_data, f)
+
+__log = logging.getLogger("chaosbot")
+
+'''
+Command Syntax
+/vote close closes issue when no nay reactions on this comment are added within voting window
+/vote reopen reopens issue when see above ^^^
+/vote label=<LABEL_TEXT> adds label when ^^^
+/vote remove-label=<LABEL_TEXT> removes label when ^^^
+/vote assign=<USER> assigns to user when ^^^
+/vote unassign=<USER> unassigns from user when ^^^
+'''
+
+COMMAND_LIST = ["/vote"]
+
+
+def can_run_vote_command(api, votes, comment_id):
+    json_data = {}
+    with open(SAVED_COMMANDS_FILE, 'r') as f:
+        json_data = json.load(f)
+
+    # Already ran this command
+    for ids_ran in json_data["comment_ids_ran"]:
+        if comment_id == ids_ran["comment_id"]:
+            __log.debug("Already ran commmand")
+            return False
+
+    # Voting window has passed
+    now = arrow.utcnow()
+    voting_window = gh.voting.get_voting_window(now)
```

Comment:
Has been removed. I think it can be replace with: `voting_window = gh.voting.get_extended_voting_window(api, settings.URN)`?